module-telemetry/src/test/java/com/mapbox/maps/module/telemetry/MapTelemetryTest.kt

```diff
@@ -32,49 +76,53 @@ class MapTelemetryTest {
     val display = mockk<Display>(relaxUnitFun = true)
     every { windowManager.defaultDisplay } returns display
 
-    mapboxTelemetry = mockk(relaxed = true)
-    telemetry = MapTelemetryImpl(mapboxTelemetry, context, "sk.foobar")
+    eventsService = mockk()
+    telemetryService = mockk()
+    telemetry = MapTelemetryImpl(context, "sk.foobar", eventsService, telemetryService)
+    every { eventsService.sendEvent(any(), any()) } returns Unit
+    mockkStatic(EventsService::class)
+    mockkStatic(TelemetryUtils::class)
+    every { TelemetryUtils.setEventsCollectionState(any(), any()) } returns Unit
   }
 
   @Test
   fun testSetUserTelemetryRequestStateEnabled() {
     telemetry.setUserTelemetryRequestState(true)
-    verify { mapboxTelemetry.enable() }
+    verify { TelemetryUtils.setEventsCollectionState(true, any()) }
   }
 
   @Test
   fun testSetUserTelemetryRequestStateDisabled() {
     telemetry.setUserTelemetryRequestState(false)
-    verify { mapboxTelemetry.disable() }
+    verify { TelemetryUtils.setEventsCollectionState(false, any()) }
   }
 
   @Test
   fun testDisableTelemetrySession() {
     telemetry.disableTelemetrySession()
-    verify { mapboxTelemetry.disable() }
+    verify { TelemetryUtils.setEventsCollectionState(false, any()) }
   }
 
   @Test
   fun testSetSessionIdRotationInterval() {
-    telemetry.setSessionIdRotationInterval(22)
-    verify { mapboxTelemetry.updateSessionIdRotationInterval(any()) }
+    assertFalse(telemetry.setSessionIdRotationInterval(22))
   }
 
   @Test
+  @Ignore("testSetDebugLoggingEnabled has been deprecated")
   fun testSetDebugLoggingEnabled() {
-    telemetry.setDebugLoggingEnabled(true)
-    verify { mapboxTelemetry.updateDebugLoggingEnabled(true) }
+    // no-ops
```

Comment:
Verify it's no-op?

Reply:
telemetry and  mapboxTelemetry has been removed from the dependency and  `setDebugLoggingEnabled` is removed from codebase, so there wouldn't be a way to validate the no-op 🤷‍♂️

Reply:
Ah, so method is not available at all, gotcha